felix/fv/rpf_test.go

```diff
@@ -186,12 +222,12 @@ var _ = infrastructure.DatastoreDescribe(
 			})
 		})
 
-		Context("With BPFEnforceRPF=Disabled", func() {
+		Context("With BPFEnforceRPF=Loose", func() {
 			BeforeEach(func() {
-				options.ExtraEnvVars["FELIX_BPFEnforceRPF"] = "Disabled"
+				options.ExtraEnvVars["FELIX_BPFEnforceRPF"] = "Loose"
 			})
 
-			It("should allow packets from wrong direction with non-strict RPF on main device", func() {
+			It("should allow packets from wrong direction with loose RPF on main device", func() {
```

Comment:
What we need to exercise with `loose` is not only that you can accept a packet from any wrong direction, but it should drop packets if there is no route back.

After testing with fakeWorkloadIP := "10.65.15.15" OK we should remove a default route from the felix (that should be ok) as that would satisfy Loose for any source. And then we should test again and the packet should be dropped because there is no route back.